auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -13,24 +12,18 @@
 class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
+    @api.model
     @tools.ormcache(skiparg=0)
```

Comment:
OK so I'm a bit confused here & I am pretty sure this is what is causing the current error.

What are we going for with this cache? Do you maybe have an example of some other code that does something similar, or would mind giving a quick breakdown for me? My google-fu is turning up zilch here.